test/bsearch/main.c

```diff
@@ -0,0 +1,80 @@
+#include "lib/lib.h"
+#include "test/test.h"
+
+static int cmp_gt(void* lhs, void* rhs)
+{
+	int l = *(int*)lhs, r = *(int*)rhs;
+	if (l == r)
+		return 0;
+	else if (l > r)
+		return 1;
+	else
+		return -1;
+}
+
+static int cmp_lt(void* lhs, void* rhs)
+{
+	int l = *(int*)lhs, r = *(int*)rhs;
+	if (l == r)
+		return 0;
+	else if (l < r)
+		return 1;
+	else
+		return -1;
+}
+
+TEST_GROUP(test_bsearch);
+
+TEST_SETUP(test_bsearch)
+{
+}
+
+TEST_TEAR_DOWN(test_bsearch)
+{
+}
+
+TEST(test_bsearch, basic)
+{
+	int arr_inc[] = {1, 2, 3, 4, 5, 6, 7, 8};
```

Comment:
`const int` (the same apples everywhere else)

Reply:
better `static const int`